Volume (#6)
MeteoSwiss/opendata-forecast-data: README.md

MeteoSwiss uses two models, **ICON-CH1-EPS** and **ICON-CH2-EPS**, to forecast atmospheric changes in Switzerland and its surroundings over a longer period than nowcasting, providing predictions for up to five days. Both models include
[ensemble data assimilation](https://www.meteoswiss.admin.ch/weather/warning-and-forecasting-systems/icon-forecasting-systems/ensemble-data-assimilation.html).

> We will cover:

Reviewer: Rather neutral than "we" for documentation, personal opinion, you can choose
```suggestion
The documentation covers the following topics:
```

### 2.3 Accessing Forecast Data

> The user can access the forecast model data from the last 24 hours. Data older than 24 hours is no longer available. The data in each collection is described in the [table](###2.1-model-specification) above.

Reviewer: ```suggestion
Users can access forecast model data from the last **24 hours**. Data older than this is no longer available. The data in each collection is described in the [Model Specification table](###2.1-model-specification).
```

> #### 2.3.1 Volume of the Forecast Data

Reviewer: ```suggestion
#### 2.3.1 Forecast Data Volume
```

> The following tables summarize the volumes of the differnt forecast files.

Reviewer: typo and small improvements (up to you)
```suggestion
The following tables summarize the volume of the different forecast files for **ICON-CH1** and **ICON-CH2**.
```

> **ICON-CH1:**

Reviewer: ```suggestion
**ICON-CH1 Data Volume**
```

> | | Single Level| Multi Level|

Reviewer: ```suggestion
| | Single-Level Files| Multi-Level Files|
```

|-----------|------------------|-----------------|
| Deterministic| 2.1 - 2.2 MiB| 74.5 - 177.4 MiB|

> | Preturbed | 21.9 - 22. 5 MiB | 1.3 - 1.7 GiB |

Reviewer: typo
```suggestion
| Perturbed | 21.9 - 22. 5 MiB | 1.3 - 1.7 GiB |
```

> **ICON-CH2:**

Reviewer: ```suggestion
**ICON-CH2 Data Volume**
```

> | | Single Level| Multi Level|

Reviewer: ```suggestion
| | Single-Level Files| Multi-Level Files|
```

|-----------|------------------|-----------------|
| Deterministic| 509.2 - 558.0 KiB| 17.9 - 43.9 MiB|

> | Preturbed | 10.0 - 10-9 MiB | 360.9 - 877.5 MiB |

Reviewer: typo
```suggestion
| Perturbed | 10.0 - 10-9 MiB | 360.9 - 877.5 MiB |
```
